# Fastqc
## Summary ##

General outline of commands and workflow for generating fastqc files from RNAseq demuxxed files on Unix command line. For small number of samples, data processed through the command line without a shell script. Fastqc files contain HTML files with several quality control metrics to assess the sequencing library.
## Process ##
    1. Connecting to lab server and moving to the right directory.
    2. Downloading sequencing data and unzipping files.
    3. Accessing fastq files and unzipping them. 
    4. Processing fastq files with fastqc program.
    5. Generating a new folder and copying fastqc files to home drive.
    
Sequencing quality can then be assessed through the HTML outputs.
   
